example_app_generator/spec/verify_custom_renderers_spec.rb

```diff
@@ -38,6 +38,7 @@ def index
       end
 
       it "renders an empty string" do
+        skip if Rails::VERSION::STRING.to_f >= 6.0
```

Comment:
Could this be inlined into the skip metadata?

Reply:
Good idea.
Done in https://github.com/rspec/rspec-rails/pull/2071/commits/86db9eb72fd42a24db3664cd5c4735b84dcfa59d

I also dig into that test and it is weird. The view is rendered even if `render_views` is not present.
I think we should try to remove the skip on the next release of rails.